azure-devops/run_build.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  targetPlatform: 'x64'
```

Comment:
Why have a `parameters` block here when the parameters come from `azure-pipelines.yml`?

Reply:
Here is declaration and default value of accepted params. The actual value is indeed in a-p.yml

Reply:
Rephrasing: do we need to have this declaration here, or can we leave it out so this template will fail when it's not used correctly? It would be a shame if someone accidentally introduced a typo in the parameter names in `azure-pipelines.yml` and we build `x64` four times for every PR.

Reply:
I suppose if it MUST be here we could give it an invalid value like `potato`.